ui/src/common/SongDetails.js

```diff
@@ -33,6 +39,13 @@ export const SongDetails = (props) => {
     updatedAt: <DateField record={record} source="updatedAt" showTime />,
     playCount: <TextField record={record} source="playCount" />,
     comment: <MultiLineTextField record={record} source="comment" />,
+    bpm: (
+      <FunctionField
+        record={record}
+        source="bpm"
+        render={(r) => r.bpm || '?'}
```

Comment:
Any reason why show a "?" when the fiend is empty? Why not do the same as `discSubtitle` and `comment`, remove the field?

Reply:
Definitely missed that it does remove those fields dynamically. Will remove it instead.